modules/ossm-deploying-multiple-control-planes.adoc

```diff
@@ -0,0 +1,181 @@
+// Module included in the following assemblies:
+// install/ossm-deploying-multiple-service-meshes-on-single-cluster.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-deploying-multiple-control-planes_{context}"]
+= Deploying multiple control planes
+:context: ossm-deploying-multiple-control-planes
+
+This procedure explains how to configure a cluster to host two control planes, each managing independent service mesh data planes. The configuration involves creating separate `{istio}` resources with unique names in independent {istio} system namespaces. Each `{istio}` resource will use a unique `revision` name, which identifies the control planes, workloads, or namespaces associated with it. These `revision` names, applied using injection or `istio.io/rev` labels determine which control plane will inject the sidecar proxy into the application pods.
+
+Each `{istio}` resource must also configure `discoverySelectors` to specify which namespaces the {istio} control plane observes. Only namespaces with labels that match the configured `discoverySelectors` can join the mesh. Additionally, discovery selectors determine which control plane creates the `istio-ca-root-cert` config map in each namespace, which is used to encrypt traffic between services with mutual TLS within each mesh.
+
+When adding an additional {istio} control plane to a cluster with an existing control plane, ensure that the existing `{istio}` instance has `discoverySelectors` configured to avoid overlapping with the new control plane. Run the following command to check for existing `{istio}` instances.
+
+This procedure may be extended to support more than two control planes. The primary limitation on the number of service meshes in a single cluster is the availability of cluster resources.
+
+[NOTE]
+====
+There is only one `IstioCNI` resource shared by all control planes in the cluster, and it must be updated independently.
+====
+
+.Prerequisites
+* The {SMProduct} operator is installed.
+* An {istio} CNI resource is created.
+* The `istioctl` binary is installed on your localhost.
+
+.Procedure
+
+. Create the namespace for the first {istio} control plane called `istio-system-1`:
++
+[source,terminal]
+----
+$ oc create namespace istio-system-1
+----
+
+. Add the following label to the first namespace, which will be used with the Istio `discoverySelector`:
```

Comment:
Not sure if formatting `discoverySelector` as code is correct, since there is no field with that exact name. The actual field name is `discoverySelectors`, so I think you either need to change this to either:
> ...with the Istio discovery selector:

or:

> ...with the Istio `discoverySelectors` field:

Reply:
Also check for other occurrences of "discoverySelector" in the rest of this document.